app/src/Homepage/Homepage-components.tsx

```diff
@@ -22,10 +43,15 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
 
 
 export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {   
+    const navigate = useNavigate();
     
     const handleClick = () => {
         console.log('Button clicked: ' + classlevel);
-        
+        if (classlevel === 'Home') {
+            navigate('/rate-my-cse/');
+        } else {
+            navigate('/rate-my-cse/cse' + classlevel.split(" ", 2)[1]);
```

Comment:
Can you make this logic more readable by any chance? It doesn't seem intuitive what this is doing. If not, can you add a comment explaining the logic?